arbnode/node.go

```diff
@@ -454,13 +454,23 @@ var ConfigDefault = Config{
 }
 
 func ConfigDefaultL1Test() *Config {
-	config := ConfigDefault
+	config := ConfigDefaultL1NonSequencerTest()
 	config.Sequencer = TestSequencerConfig
-	config.L1Reader = headerreader.TestConfig
-	config.InboxReader = TestInboxReaderConfig
 	config.DelayedSequencer = TestDelayedSequencerConfig
 	config.BatchPoster = TestBatchPosterConfig
 	config.SeqCoordinator = TestSeqCoordinatorConfig
+
+	return config
+}
+
+func ConfigDefaultL1NonSequencerTest() *Config {
+	config := ConfigDefault
+	config.L1Reader = headerreader.TestConfig
+	config.InboxReader = TestInboxReaderConfig
+	config.Sequencer.Enable = false
+	config.DelayedSequencer.Enable = false
+	config.BatchPoster.Enable = false
+	config.SeqCoordinator.Enable = false
```

Comment:
Maybe consider not setting these or panicking if they're not set to false since I think we want to make sure that they're not. Even better might be to write a separate test to just check the defaults